### DISCUSSION TITLE
Any plans for supporting flathub rpm based client or snap?

### DISCUSSION THREAD

**Discussion by doxxedd** [Nuzair46/BlockTheSpot-Linux#30] [Q&A]
Just wanted to know what options there are if any for Fedora
Thanks

**Comment by Myron2812** [comment]
Are you also struggling to find the path on Fedora? If so, I found it a couple days back thanks to the spicetify config file. Here you go:
```bash
/var/lib/flatpak/app/com.spotify.Client/x86_64/stable/active/files/extra/share/spotify/
